pages/receive/CreateInvoice.tsx

```diff
@@ -0,0 +1,200 @@
+import { Nip47Transaction } from "@getalby/sdk/dist/NWCClient";
+import * as Clipboard from "expo-clipboard";
+import { router } from "expo-router";
+import React from "react";
+import { Image, View } from "react-native";
+import Toast from "react-native-toast-message";
+import DismissableKeyboardView from "~/components/DismissableKeyboardView";
+import { DualCurrencyInput } from "~/components/DualCurrencyInput";
+import { Copy } from "~/components/Icons";
+import Loading from "~/components/Loading";
+import QRCode from "~/components/QRCode";
+import Screen from "~/components/Screen";
+import { Button } from "~/components/ui/button";
+import { Input } from "~/components/ui/input";
+import { Text } from "~/components/ui/text";
+import { useGetFiatAmount } from "~/hooks/useGetFiatAmount";
+import { errorToast } from "~/lib/errorToast";
+import { useAppStore } from "~/lib/state/appStore";
+
+export function CreateInvoice() {
```

Comment:
This page does more than just creating an invoice, is there a better name for it?